srcpkgs/python3-py7zr/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-py7zr'
+pkgname=python3-py7zr
+version=0.11.3
+revision=1
+wrksrc="py7zr-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip python3-pycryptodome python3-pyannotate"
+depends="python3-pycryptodome python3-texttable"
+short_desc="Pure python 7-zip library"
+maintainer="skmpz <dem.procopiou@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://github.com/miurahr/py7zr"
+distfiles="${PYPI_SITE}/p/py7zr/py7zr-${version}.tar.gz"
+checksum=f9e872f02daf9be64ef35a0d69f5467a3ccbd028df0863ee39a2fbab5f80cc52
+# check if tests can be fixed
+make_check=no
```

Comment:
How have you tried to get tests working?

Reply:
well.. it seems that a lot of packages needed for tests (pytest-timeout, pytest-benchmark etc.) are missing.. Should we try to get all those merged as well ?

Reply:
In that case, the comment should mention that we lack those dependencies :)